### ISSUE TITLE
SERVER ERROR: Cannot read property 'map' of undefined

### ISSUE THREAD

**Issue by KwakesProject** [angular/vscode-ng-language-service#78]
Hi. As soon as VSCode updated to Angular Language Service v0.1.3, I got the following error in the output termianl:

```
Search path: c:/Users/Kwakes/Documents/Dev/HolitionCMS/cms-frontend/src/app/views/client/client-list
Config file name: c:/Users/Kwakes/Documents/Dev/HolitionCMS/cms-frontend/tsconfig.json
Angular Language Service: 4.1.0-beta.0-96aa236
TypeScript: 2.1.6
SERVER ERROR: Cannot read property 'map' of undefined
TypeError: Cannot read property 'map' of undefined
    at Evaluator.evaluateNode (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js:42680:54)
    at _loop_1 (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js:43435:57)
    at C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js:43495:25
    at visitEachNode (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\typescript\lib\typescript.js:13888:30)
    at Object.forEachChild (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\typescript\lib\typescript.js:14059:24)
    at MetadataCollector.getMetadata (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js:43310:14)
    at ReflectorHost.CompilerHost.getMetadataFor (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js:43908:51)
    at StaticSymbolResolver.getModuleMetadata (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js:40131:62)
    at StaticSymbolResolver._createSymbolsOf (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js:39935:46)
    at StaticSymbolResolver.getSymbolsOf (C:\Users\Kwakes\.vscode\extensions\Angular.ng-template-0.1.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js:39916:14)
[Error - 17:00:23] Notification handler 'textDocument/didSave' failed with message: Cannot read property 'map' of undefined

```

this has only happened on my work PC and not my macbook pro

**Comment by chuckjaz** [comment]
It looks like the language service is receiving a `new` expression that doesn't have any `arguments` in the AST (even if it doesn't have arguments it should have an empty arguments property).

Can you send create a repository I can clone or a .zip file that demonstrates this issue? If not can you try to narrow it down knowing that it is probably some `new` expression in your project that is throwing off the language service?

**Comment by chuckjaz** [comment]
I was able to reproduce this.

Tracking in Angular https://github.com/angular/angular/issues/15906

**Comment by KwakesProject** [comment]
sorry for the delay. didn't check my emails. this issue is only on one project i'm working on. it a work project so a private repo. 
did you manage to reproduce the issue or do you need me to try and send a copy of the code

**Comment by chuckjaz** [comment]
I reproduced it and have fix pending in the angular repository.

**Comment by chuckjaz** [comment]
Fixed in 0.1.4
